readme.md

```diff
@@ -1,4 +1,13 @@
 # Marvin
+
 The new Wikipedia mobile website. Read more on our [wiki]!
 
-[wiki]: https://www.mediawiki.org/wiki/Reading/Web/Projects/NewMobileWebsite
\ No newline at end of file
+## Development
+
+Marvin requires the Node.js active LTS version, and npm > 5.X.
```

Comment:
How about we just add a `"engines": { "node": "^5.0.0" }` to package.json and remove this line?